src/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework.UnitTests/ExampleTests.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.LanguageServer;
```

Comment:
This is to use the NoOpLspLogger that already existed in this namespace from Microsoft.CodeAnalysis.LanguageServer.Protocol. If there is a desire that this example _must_ use only stuff from CommonLanguageServerProtocol, then we should delete the package reference and move the no-op implementation to CLaSP.